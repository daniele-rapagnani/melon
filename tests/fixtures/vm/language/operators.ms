io.print(2^24);
io.print(2 % 2);
io.print(3 % 2);
io.print((1 + 1)^(10 * 2 + 4));
io.print(4.0^0.5);

if (10 < 5 + 6) {
    io.print("10 < 5 + 6");
}

let a = 4;

for (let i in a+1..10) {
    io.print(i);
} 