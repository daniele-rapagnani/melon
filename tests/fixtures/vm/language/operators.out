16777216
0
1
16777216
2.000000000
10 < 5 + 6
5
6
7
8
9
